### PR TITLE
Support RBAC management for instances

### PR DESCRIPTION
**Pull request by TheBigLee**
## Summary
* This PR adds support to specify custom groups and users that should have access to the instance namespace.

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [ ] Update tests.
- [ ] Link this PR to related issues.

### CODE REVIEW

File: pkg/comp-functions/functions/common/instance_namespace.go [hunk 1]
```diff
@@ -139,7 +139,7 @@ func createInstanceNamespace(serviceName, compName, claimNamespace, instanceName
 	return svc.SetDesiredKubeObjectWithName(ns, instanceNamespace, namespaceResName)
 }
 
-func createNamespacePermissions(instance string, instanceNs string, namespaceResName string, svc *runtime.ServiceRuntime) error {
+func createNamespacePermissions(instance string, instanceNs string, namespaceResName string, comp Composite, svc *runtime.ServiceRuntime) error {
```
Comment (Kidswiss) [inline pkg/comp-functions/functions/common/instance_namespace.go:142]: We could just accept the `Security` object directly here.

File: pkg/comp-functions/functions/common/instance_namespace.go [hunk 2]
```diff
@@ -158,7 +158,38 @@ func createNamespacePermissions(instance string, instanceNs string, namespaceRes
 		return fmt.Errorf("cannot get claim namespace: %w", err)
 	}
 
-	if org == "" {
+	security := comp.GetSecurity()
```
Comment (Kidswiss) [inline pkg/comp-functions/functions/common/instance_namespace.go:161]: Then you can avoid this line.

**Review by Kidswiss — CHANGES_REQUESTED** [review]
Just a small nit

File: pkg/comp-functions/functions/common/instance_namespace.go (hunk 1 above)
Comment (TheBigLee) [reply pkg/comp-functions/functions/common/instance_namespace.go]: True. This makes more sense.

File: pkg/comp-functions/functions/common/instance_namespace.go (hunk 2 above)
Comment (TheBigLee) [reply pkg/comp-functions/functions/common/instance_namespace.go]: I've removed it

**Review by Kidswiss — APPROVED** [review]

**Pull request merged**